vendor/knative.dev/pkg/profiling/server.go

```diff
@@ -25,16 +25,12 @@ import (
 	perrors "github.com/pkg/errors"
 	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis/networking"
 )
 
-const (
-	// profilingPort is the port where we expose profiling information if profiling is enabled
-	profilingPort = ":8008"
-
-	// profilingKey is the name of the key in config-observability config map that indicates whether profiling
-	// is enabled of disabled
-	profilingKey = "profiling.enable"
-)
+// profilingKey is the name of the key in config-observability config map that indicates whether profiling
+// is enabled of disabled
```

Comment:
`enabled of disabled` Is this typo?

Reply:
Ah, the change under `vendor` does not need to review as per https://github.com/knative/serving/pull/5277/files#r317714150 ?